pulsar-broker/src/main/java/org/apache/pulsar/broker/service/BrokerService.java

```diff
@@ -1918,6 +1918,9 @@ private void createPendingLoadTopic() {
     }
 
     public CompletableFuture<PartitionedTopicMetadata> fetchPartitionedTopicMetadataCheckAllowAutoCreationAsync(TopicName topicName) {
+        if(pulsar.getNamespaceService() == null){
+            return CompletableFuture.completedFuture(null);
```

Comment:
We should return an error in the future at this point, otherwise we're moving the NPE somewhere else

Reply:
OK